dotnet/src/SemanticKernel.Core/Data/TextSearch/TextSearchExtensions.cs

```diff
@@ -243,9 +460,29 @@ private static KernelFunctionFromMethodOptions DefaultGetTextSearchResultsMethod
             ReturnParameter = new() { ParameterType = typeof(KernelSearchResults<TextSearchResult>) },
         };
 
+    /// <summary>
+    /// Create the default <see cref="KernelFunctionFromMethodOptions"/> for <see cref="ITextSearch.GetTextSearchResultsAsync(string, TextSearchOptions?, CancellationToken)"/>.
+    /// <param name="jsonSerializerOptions">The <see cref="JsonSerializerOptions"/> used for generating JSON schema for method parameters and return type.</param>
+    /// </summary>
+    private static KernelFunctionFromMethodOptions DefaultGetTextSearchResultsMethodOptions(JsonSerializerOptions jsonSerializerOptions) =>
+        new()
+        {
+            FunctionName = "GetTextSearchResults",
+            Description = "Perform a search for content related to the specified query. The search will return the name, value and link for the related content.",
+            Parameters =
```

Comment:
Would it make sense to move the parameter list into a static private field, since it's repeated a lot?

Reply:
It makes sense, done.